test/language/expressions/typeof/proxy.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2019 Ecma International. All rights reserved.
```

Comment:
This is fine and correct for the Copyright's PoV, but I believe you should use your name in the next tests.